### DISCUSSION TITLE
Having title and citekey in filename for Mdnotes

### DISCUSSION THREAD

**Discussion by mhofherr** [argenos/zotero-mdnotes#103] [Q&A]
Hi all,

inspired by the good Howtos in the Obsidian Forum I finally shifted my Zotero-output to zotfile/mdnotes. Only one remaining question: Is it possible to use variables in the output-filenames for Mdnotes? 
It seems I can either use the title as filename or the citekey, enhanced by prefix/suffix. Just using the citekey makes linking with hundreds of books/whitepapers a little bit awkward in Obsidian, and using just the title lacks some ... uniqueness.
My currently preferred solution would be to use the title and add the citekey as prefix. Is that possible? Prefix seems to support only fixed strings.  

Any other ideas/recommendations to combine title and citeky in the filename?

Thanks,
Matthias

**Comment by argenos** [comment]
Not right now, but subscribe to this issue for updates: https://github.com/argenos/zotero-mdnotes/issues/94
